packages/core/src/shared/ui/sam/bucket.ts

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { DefaultS3Client } from '../../clients/s3Client'
+import { samDeployUrl, samSyncUrl } from '../../constants'
+import { createCommonButtons } from '../buttons'
+import { createQuickPick, DataQuickPickItem } from '../pickerPrompter'
+import type { SyncParams } from '../../sam/sync'
+import * as nls from 'vscode-nls'
+import { getRecentResponse } from '../../sam/utils'
+
+const localize = nls.loadMessageBundle()
```

Comment:
I would suggest giving a more descriptive naming to this file, the current naming gives impression that the file create a bucket resource. Maybe name it to `bucket-prompter`

Reply:
I'll update the name.